ui_framework/src/components/icon/_icon.scss

```diff
@@ -1,6 +1,11 @@
 @include component('kuiIcon') {
+  opacity: .8;
   display: inline-block;
 
+  &:hover {
+    opacity: 1;
+  }
```

Comment:
I don't think we want to couple interactive behavior so tightly with a primitive component like an icon since we could use icons in all kinds of places and they won't always be interactive. Seems like this behavior belongs in the kuiHeaderButton?